v0/destinations/singular/util.js

```diff
@@ -138,6 +139,26 @@ const platformWisePayloadGenerator = (message, isSessionEvent) => {
     } else {
       payload.dnt = 1;
     }
+
+    // Singular accepts install_time & update_time values in Unix timestamps format,
+    // hence we are converting it
+    const it =
+      message.properties?.install_time ||
+      message.originalTimestamp ||
+      message.timestamp;
+    const ut =
+      message.properties?.update_time ||
+      message.originalTimestamp ||
+      message.timestamp;
+    payload.install_time = toUnixTimestamp(it);
+    payload.update_time = toUnixTimestamp(ut);
```

Comment:
Why cant we do this using mapping json. We can have this `toUnixTimestamp` as metadata function in mapping json itself right?

Reply:
reinstated mapping and used metadata

Reply:
Not seeing can you push your changes